src/AppExample.tsx

```diff
@@ -8,87 +8,81 @@ import UseErrorDispatcherExample from './examples/UseErrorDispatcherExample';
-        sx={{
-          display: 'flex',
-          flexDirection: 'column',
-          minHeight: '100vh',
-        }}
-      >
-        <Header withSecondHeader={false} onExitAction={onLogout} />
-        <UserNotifyHandle />
-        <LoadingOverlay />
-        <UnloadEventHandler />
+const AppExample = () => (
+  <ErrorBoundary assistanceEmail="assistenza@selfcare.it">
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        minHeight: '100vh',
+      }}
+    >
+      <HeaderExample />
+      <UserNotifyHandle />
+      <LoadingOverlay />
+      <UnloadEventHandler />
 
-        <TitleBox title="Title example" subTitle="Subtitle example" />
+      <TitleBox title="Title example" subTitle="Subtitle example" />
 
-        <Grid container direction="row">
-          <CustomAvatarExample />
+      <Grid container direction="row">
+        <CustomAvatarExample />
 
-          <Grid item xs={6}>
-            <CustomPaginationExample />
-          </Grid>
+        <Grid item xs={6}>
+          <CustomPaginationExample />
         </Grid>
+      </Grid>
 
-        <Grid container direction="row" flexGrow={1} spacing={1} mt={1}>
-          <Grid item xs={1}>
-            <UseLoadingExample />
-          </Grid>
+      <Grid container direction="row" flexGrow={1} spacing={1} mt={1}>
+        <Grid item xs={1}>
+          <UseLoadingExample />
+        </Grid>
 
-          <Grid item xs={8}>
-            <UseErrorDispatcherExample />
-          </Grid>
+        <Grid item xs={8}>
+          <UseErrorDispatcherExample />
+        </Grid>
 
-          <Grid item xs={1}>
-            <FilterModalExample />
-          </Grid>
+        <Grid item xs={1}>
+          <FilterModalExample />
+        </Grid>
 
-          <Grid item xs={1}>
-            <SessionModalExample />
-          </Grid>
+        <Grid item xs={1}>
+          <SessionModalExample />
+        </Grid>
 
-          <Grid item xs={1}>
-            <ToastExample />
-          </Grid>
+        <Grid item xs={1}>
+          <ToastExample />
+        </Grid>
 
-          <Grid item xs={4}>
-            <UseUserNotifyExample />
-          </Grid>
+        <Grid item xs={4}>
+          <UseUserNotifyExample />
+        </Grid>
 
-          <Grid item xs={3}>
-            <UseUnloadEventInterceptorExample />
-          </Grid>
+        <Grid item xs={3}>
+          <UseUnloadEventInterceptorExample />
+        </Grid>
 
-          <Grid item xs={2}>
-            <AnalyticsExample />
-          </Grid>
-          <Grid item xs={2}>
-            <TranslationTextExample />
-          </Grid>
+        <Grid item xs={2}>
+          <AnalyticsExample />
+        </Grid>
+        <Grid item xs={2}>
+          <TranslationTextExample />
         </Grid>
-        <Footer assistanceEmail="assistenza@selfcare.it" onExit={onExit} />
-      </Box>
-    </ErrorBoundary>
-  );
-};
+      </Grid>
+      {/* if false, show more links */}
+      <Footer loggedUser={true} />
```

Comment:
make a FooterExample which render a checkbox or a button or a toggle, which allow to change between the two behavior